index.html

```diff
@@ -3,28 +3,53 @@
 <head>
     <meta charset="UTF-8">
     <link href="css/style.css" rel="stylesheet">
-    
-    <!-- Bootstrap CSS-->
+
+    <!-- Bootstrap CSS -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
 
-    <link rel="apple-touch-icon" sizes="1831x1831" href="/smartplan.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="/smartplan.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="/fsmartplan.png">
+    <!-- Icons -->
+    <link rel="apple-touch-icon" sizes="1831x1831" href="img/smartplan.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="img/smartplan.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="img/smartplan.png">
+
+    <!-- Primary Meta Tags -->
+    <title>SMART PLAN - BSZET</title>
+    <meta name="title" content="SMART PLAN - BSZET" />
+    <meta name="description" content="ORGANIZING WITH EASE" />
     <meta name="msapplication-TileColor" content="#e8ecef">
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="application-name" content="SMART PLAN">
     <meta name="apple-mobile-web-app-title" content="SMART PLAN">
-    <meta name="theme-color" content="#e8ecef">
     <meta name="msapplication-navbutton-color" content="#e8ecef">
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
     <meta name="msapplication-starturl" content="/">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <!-- Primary Meta Tags -->
-    <title>SMART PLAN - BSZET</title>
-    <meta name="title" content="SMART PLAN - BSZET" />
-    <meta name="description" content="ORGANIZING WITH EASE" />
+    <style>
+        body {
+            display: flex;
+            flex-direction: column;
+            min-height: 100vh;
+            margin: 0;
+        }
+
+        main {
+            flex: 1;
+        }
+
+        footer {
+            font-size: 70%;
+        }
+
+        footer a, a:hover, a:focus, a:active {
+            text-decoration: none;
+            color: inherit;
+            /* also all:unset; possible but not sure if it works on old browsers */
+        }
+
+        
+    </style>
```

Comment:
Please move that to the .css file, messy to have .css in both the index.html and a style.css.

Reply:
you're right, no idea why it's in here, tx

Reply:
Update it please, then mark as resolved.